lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -842,6 +842,9 @@ def get_options_with_changing_values(client, module, parameters):
     apply_immediately = parameters.pop('ApplyImmediately', None)
     cloudwatch_logs_enabled = module.params['enable_cloudwatch_logs_exports']
 
+    # Iops can't be updated
+    parameters.pop('Iops', None)
```

Comment:
Actually this only becomes an issue when providing the iops param without setting `storage_type:` to `io`. Hence we should only update docs and better catch the errors by checking provided params compatibility